project.clj

```diff
@@ -0,0 +1 @@
+(defproject tolitius/verter "dev-placeholder")
```

Comment:
This allows dev work with leiningen checkouts.

Reply:
let's not add more deps / build tools artifacts
lein, boot, deps.edn, tools.deps, tools.build, etc..

feel free to add it locally when if you need it

Reply:
Will do.